guide/src/telemetry.md

```diff
@@ -35,9 +35,9 @@ The following table describes the metrics currently tracked by the telemetry ser
 | `workers`                    | Number of workers per object                         | `i64` UpDownCounter |
 | `ibc_client_updates`         | Number of client updates performed per client        | `u64` Counter       |
 | `ibc_client_misbehaviours`   | Number of misbehaviours detected per client          | `u64` Counter       |
-| `ibc_receive_packets`        | Number of receive packets relayed per channel        | `u64` Counter       |
-| `ibc_acknowledgment_packets` | Number of acknowledgment packets relayed per channel | `u64` Counter       |
-| `ibc_timeout_packets`        | Number of timeout packets relayed per channel        | `u64` Counter       |
+| `ibc_receive_packets`        | Number of confirmed receive packets relayed per channel. Available if relayer runs with Tx confirmation enabled        | `u64` Counter       |
+| `ibc_acknowledgment_packets` | Number of confirmed acknowledgment packets relayed per channel. Available if relayer runs with Tx confirmation enabled | `u64` Counter       |
+| `ibc_timeout_packets`        | Number of confirmed timeout packets relayed per channel. Available if relayer runs with Tx confirmation enabled        | `u64` Counter       |
 | `wallet_balance`             | The balance of each wallet Hermes uses per chain. Please note that when converting the balance to f64 a loss in
```

Comment:
I have this broken output for this (see the `\n`, looks weird in the browser)
\# HELP wallet_balance The balance of each wallet Hermes uses per chain. Please note that when converting the balance to f64 a loss in\n                precision might be introduced in the displayed value